src/components/gltf-model-plus.js

```diff
@@ -591,6 +592,36 @@ class GLTFMozTextureRGBE {
   }
 }
 
+class GLTFMozTextureEXR {
+  constructor(parser, loader) {
+    this.parser = parser;
+    this.loader = loader;
+    this.name = "MOZ_texture_exr";
+  }
+
+  loadTexture(textureIndex) {
+    const parser = this.parser;
+    const json = parser.json;
+    const textureDef = json.textures[textureIndex];
+
+    if (!textureDef.extensions || !textureDef.extensions[this.name]) {
+      return null;
+    }
+
+    const extensionDef = textureDef.extensions[this.name];
+    const source = json.images[extensionDef.source];
+    return parser.loadTextureImage(textureIndex, source, this.loader).then(t => {
+      // TODO pretty severe artifacting when using mipmaps, disable for now (assumption copied from GLTFMozTextureRGBE)
```

Comment:
I suspect this was to do with decoding the RGBM which shouldn't be a factor here, the mips should blend more smoothly here since EXR actually uses float values from the start. Didn't test it obviously.

Reply:
Makes sense. I'll remove and check.